client/src/components/SortDirection/SortDirection.stories.js

```diff
@@ -0,0 +1,19 @@
+import _ from "lodash";
+import React from "react";
+
+import { SortDirection } from "./SortDirection";
+
+export default {
+  title: "SortDirection",
+  component: SortDirection,
+};
+
+const Template = (args) => <SortDirection {...args} />;
```

Comment:
Should give it a background color. Can't see the white arrows on white background!